carina-api/src/main/java/com/qaprosoft/apitools/validation/XmlValidator.java

```diff
@@ -15,13 +18,30 @@
 import org.slf4j.LoggerFactory;
 import org.xml.sax.SAXException;
 
-
 public class XmlValidator {
     private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
     private XmlValidator() {
     }
 
+    /**
+     * @param mode - determines how to compare 2 XMLs. See type description for more details.
+     */
+    public static void validateXml(String actualXmlData, String expectedXmlPath, XmlCompareMode mode) {
+        try {
+            String expectedXmlData = Files.lines(Path.of(expectedXmlPath))
+                    .collect(Collectors.joining("\n"));
+            if (mode == XmlCompareMode.NON_STRICT) {
+                XmlComparator.nonStrictOrderCompare(actualXmlData, expectedXmlData);
+            } else {
+                XmlComparator.strictCompare(actualXmlData, expectedXmlData);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
what will happen if Exception is thrown?
don't you want to change it to Runtime exception instead?